eth/api_debug_test.go

```diff
@@ -33,16 +33,15 @@ import (
 	"strings"
 	"testing"
 
+	"github.com/ava-labs/libevm/common"
+	"github.com/ava-labs/libevm/core/rawdb"
+	"github.com/ava-labs/libevm/core/types"
+	"github.com/ava-labs/libevm/crypto"
 	"github.com/ava-labs/libevm/triedb"
-	"github.com/ava-labs/subnet-evm/core/rawdb"
 	"github.com/ava-labs/subnet-evm/core/state"
-	"github.com/ava-labs/subnet-evm/core/types"
-	"github.com/holiman/uint256"
 
```

Comment:
should we delete this wspace and have it be a single import chunk like upstream?

Reply:
Let's do this in a "final changes PR" (already got a branch here, and will takes changes from https://github.com/ava-labs/coreth/pull/862 as well - will do in that coreth PR too!)

Reply:
Done on the coreth's side, and done here in draft PR for final changes #1515

Reply:
Incorporated in the last commit of this branch in this PR.